ui.apps/src/main/content/jcr_root/apps/core/cif/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -37,6 +37,23 @@
             return response.json();
         }
 
+        async _fetchGraphql(query) {
+            let params = {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({ query })
+            };
+
+            let response = await this._fetch(this.endpoint, params);
+            if (response.errors) {
```

Comment:
I remember @cjelger saying something about the `response.errors` not being the best way to check for errors since a response can contain both errors and data.

Reply:
I refactored the code here into a separate function and didn't change existing functionality. I'll take care of that in a follow up PR :)